eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/IntegerConverter.php

```diff
@@ -121,20 +121,20 @@ public function getIndexColumn()
      *   - {@link self::HAS_MAX_VALUE}
      *   - {@link self::HAS_MIN_VALUE}.
      *
-     * @param int|null $minValue Minimum int value, or null if not set
-     * @param int|null $maxValue Maximum int value, or null if not set
+     * @param int|false|null $minValue Minimum int value, false if none, or null if not set
+     * @param int|false|null $maxValue Maximum int value, false if none, or null if not set
```

Comment:
Yes, that would be the case, which brings me to think that always changing storage values will result in wrong data on partial updates (only min or max) so a bad approach...